integrations/server/test_covid_hosp.py

```diff
@@ -81,3 +81,22 @@ def insert_issue(cur, issue, value, record_type):
       self.assertEqual(rows[1]['critical_staffing_shortage_today_yes'], 456)
       self.assertEqual(rows[2]['issue'], 20201203)
       self.assertEqual(rows[2]['critical_staffing_shortage_today_yes'], 789)
+
+
+  def test_query_by_as_f(self):
+    with Database.connect() as db:
+      with db.new_cursor() as cur:
+        # inserting out of order to test server-side order by
+        # also inserting two for 20201201 to test tiebreaker.
+        self.insert_issue(cur, 20201101, 0, 'T')
+        self.insert_issue(cur, 20201102, 1, 'D')
+        self.insert_issue(cur, 20201103, 2, 'D')
+        self.insert_issue(cur, 20201103, 3, 'T')
+        self.insert_issue(cur, 20201104, 4, 'T')
+
+    with self.subTest(name='as_of with multiple issues'):
+      response = Epidata.covid_hosp('PA', 20201118, as_of=20201103)
+      self.assertEqual(response['result'], 1)
+      self.assertEqual(len(response['epidata']), 1)
```

Comment:
this is the test that will fail with the old code, since it returns a length of 3.